completion/available/nikto.completion.bash

```diff
@@ -0,0 +1,35 @@
+# shellcheck shell=bash
+
+__nikto_completion() {
+	local cur=${COMP_WORDS[COMP_CWORD]}
```

Comment:
This line isn't used anymore, so you can remove it. `shellcheck` is complaining about the unused variable.
```suggestion
```

Reply:
@gaelicWizard how did you proposed this change. This is amazing 🤩 

![image](https://user-images.githubusercontent.com/28386721/153078768-dde5a974-267b-4c81-a500-e6503636e414.png)